helpdesk/templates/helpdesk/ticket_list.html

```diff
@@ -6,32 +6,100 @@
 
 <script src='{% static "helpdesk/filter.js" %}'></script>
 <script>
-$(document).ready(function() {
+    {% if server_side %}
+        $(document).ready(function()
+        {
+          let url = ""
+            //DataTables Initialization
+            let tasks_table = $('#ticketTable').DataTable({
+                "language": {
+                    "emptyTable": "{% trans 'No Tickets Match Your Selection' %}"
+                },
+                "processing": true,
+                "serverSide": true,
+                "ajax": {
+                    "url": "/datatables_ticket_list/",
+                    "type": "GET",
+                },
+                createdRow: function( row, data, dataIndex )
+                {
+                    $( row ).addClass(data.row_class);
+                },
 
-    $('#ticketTable').DataTable({
-            "oLanguage": {
-                "sEmptyTable": "{% trans 'No Tickets Match Your Selection' %}"
-            },
-            "order": [],
-            responsive: true
-    });
-
-    $("#select_all").click(function() {
+                "columns": [
+                  {"data": "ticket",
+                    "render": function (data, type, row, meta)
+                    {
+                        var id = data.split(" ")[0];
+                        var name = data.split(" ")[1];
+                        let url = "{% url 'helpdesk:view' 1234 %}".replace(/1234/, id.toString());
+                        if (type === 'display')
+                        {
+                          data = '<b><a href="' + url + '" >' + name + '</a></b>';
+                        }
+                        return data
+                    }
+                  },
+                  {"data": "id",
+                    "orderable": false,
+                    "render": function(data, type, row, meta)
+                    {
+                      var pk = data;
+                      if(type === 'display'){
+                        data = "<input type='checkbox' name='ticket_id' value='"+pk+"'"+ "class='ticket_multi_select' />"
+                      }
+                        return data
+                      }
+                  },
+                  {"data": "priority"},
+                  {"data": "title",
+                    "render": function (data, type, row, meta)
+                    {
+                      if (type === 'display')
+                      {
+                        data = '<b><a href="' + url + '" >' + data + '</a></b>';
```

Comment:
This is not the correct url. The url set in line 35 is out of scope at this point. https://github.com/django-helpdesk/django-helpdesk/issues/681

Reply:
Merged pull #699 to fix this, thanks.